src/printer/index.js

```diff
@@ -565,6 +571,18 @@ function genericPrint(path, options, print) {
     }
 
     case "Expr": {
+      if (n.value) {
+        if (n.value.ast_type === "BoolOp" || n.value.ast_type === "BinOp") {
+          return group(
+            concat([
+              ifBreak("("),
+              indent(concat([softline, path.call(print, "value")])),
+              softline,
+              ifBreak(")")
+            ])
+          );
+        }
+      }
```

Comment:
This looks like a pretty common pattern (check if it's a `BoolOp` or `BinOp` then do this group and indent with the parens). Would it make sense to pull it out to a function?